tools/pd-ctl/pdctl/command/log_command.go

```diff
@@ -29,7 +29,7 @@ var (
 // NewLogCommand New a log subcommand of the rootCmd
 func NewLogCommand() *cobra.Command {
 	conf := &cobra.Command{
-		Use:   "log [fatal|error|warn|info|debug]",
+		Use:   "log [fatal|error|warn|info|debug] [addr]",
```

Comment:
May I set all endpoints's log  to be info.

Reply:
No, It's not easy to get all PD members with a single request.